src/utils/data-management/shareExperiment.js

```diff
@@ -0,0 +1,52 @@
+import fetchAPI from 'utils/http/fetchAPI';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import handleError from 'utils/http/handleError';
+
+const sendInvites = async (addedUsers, experimentInfo) => {
+  const {
+    experimentId, experimentName, activeProjectUuid, role,
+  } = experimentInfo;
+
+  const requests = addedUsers.map(async (user) => {
+    try {
+      await fetchAPI(`/v1/access/${experimentId}`, {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          projectUuid: activeProjectUuid,
+          role,
+          userEmail: user,
+        }),
+      });
+      pushNotificationMessage('success', `User ${user} has been successfully invited to view ${experimentName}.`);
+    } catch (e) {
+      handleError(e);
+    }
+  });
+
+  return Promise.all(requests);
+};
+
+const revokeRole = async (userEmail, experimentInfo) => {
```

Comment:
This would be better in its own file and we can call this file sendInvites instead then

Reply:
separated into own files